config/prod/gene-drug-db-test-instance-ec2.yaml

```diff
@@ -16,5 +16,5 @@ parameters:
   JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
 hooks:
   after_create:
-    - !notify_ec2
+    - !ec2_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}
```

Comment:
Wheres this 'stack_group_config' coming from?

Reply:
it's a reference to vars in `config/config.yaml` file.  more info here: https://sceptre.cloudreach.com/latest/docs/stack_group_config.html